spec/models/mediaflux/http/get_metadata_request_spec.rb

```diff
@@ -24,8 +24,6 @@
       expect(metadata[:collection]).to be_falsey
       expect(metadata[:path]).to eq("/td-test-001/collection-96-55948/file-96-57045")
       expect(metadata[:type]).to eq("content/unknown")
-      expect(metadata[:size]).to eq("")
```

Comment:
@JaymeeH I would expect the test to make sure the correct value is here...

Reply:
Gotcha, I'll work on that right now